File: year2020/month09/day0930/ms_0930.cs
using System;

namespace algorithm_68
{
    class ms_0930
    {
        public const double PI = 3.141592;

        static void Circle(double radius) 
        {
            double value = PI * radius * radius;
            
            Console.WriteLine($"area : {value:#.###}");
        }

        static void Main(string[] args)
        {
            double radius;

            Console.Write("radius : ");
            radius = Convert.ToDouble(Console.ReadLine());

            Circle(radius);
        }
    }
}

File: year2020/month09/day0930/ms_0930.cpp
#include <iostream>
#define PI 3.141592

using namespace std;

void circle(double radius);

int main() {
	double radius;

	cout << "radius : ";
	cin >> radius;

	circle(radius);

	return 0;
}

void circle(double r) {
	double value = r * r * PI;

	printf("area : %.3lf", value);
}